README.md

```diff
@@ -20,7 +20,7 @@ The Fabric 1.16.3 version is in active development.
 
 ## Installing
 
-KAMI is a fabric mod. Download the [latest release](https://github.com/zeroeightysix/KAMI/releases), or preferably, [**build it yourself**](#building).
+KAMI is a fabric mod. Download the [latest release](#downloading), or [build it yourself](#building).
```

Comment:
Why remove the `preferably`?

Reply:
because actions builds the latest commit for you, and it's a consistent build environment